### PR TITLE
Update quicktour

### PR DESCRIPTION
**Pull request by stevhliu**
This PR combines the relevant content from the general Diffusers [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/diffusers_intro.ipynb) into the quicktour to showcase the other two core API components (models and schedulers) in addition to the pipeline. It is essentially a lightweight/simplified version of the notebook aimed at getting users started as quickly as possible without necessarily having to read some of the more background and optional content (we keep a link to the notebook, so no information is lost).

### CODE REVIEW

File: docs/source/en/quicktour.mdx [hunk 1]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 ```
 
-For more in-detail information on how to change between schedulers, please refer to the [Using Schedulers](./using-diffusers/schedulers) guide.
+Try generating an image with the new scheduler and see if you notice a difference!
+
+In the next section, you'll take a closer look at the components - the model and scheduler - that make up the [`DiffusionPipeline`] and learn how to use these components to generate an image of a cat.
+
+## Models
+
+Models take a noisy sample, and at each timestep it predicts the *noise residual*, the difference between a less noisy image and the input image. You can mix and match models to create other diffusion systems.
```
Comment (yiyixuxu) [inline docs/source/en/quicktour.mdx:156]: it can predict the noise residual, but also other things - e.g. It can predict the slightly less noisy image directly in some cases. 

I think noise residual is probably the most common objective, so it's ok to only talk about that. But we probably need to point out that it's just one of the objectives so the sentence is accurate

File: docs/source/en/quicktour.mdx [hunk 2]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+...     with torch.no_grad():
+...         residual = model(sample, t).sample
+
+...     # 2. compute less noisy image and set x_t -> x_t-1
+...     sample = scheduler.step(residual, t, sample).prev_sample
+
+...     # 3. optionally look at image
+...     if (i + 1) % 50 == 0:
+...         display_sample(sample, i + 1)
+```
```
Comment (yiyixuxu) [inline docs/source/en/quicktour.mdx:293]: Do we need to add actual images here so users can see them without actually running the notebook?

File: docs/source/en/quicktour.mdx [hunk 3]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+## Models
+
+Models take a noisy sample, and at each timestep it predicts the *noise residual*, the difference between a less noisy image and the input image. You can mix and match models to create other diffusion systems.
+
+Models are initiated with the [`~ModelMixin.from_pretrained`] method which'll also locally cache the model weights so it is faster the next time you load the model. For the quicktour, you'll load the [`UNet2DModel`], a basic unconditional image generation model with a checkpoint trained on cat images:
+
+```py
+>>> from diffusers import UNet2DModel
+
+>>> repo_id = "google/ddpm-cat-256"
```
Comment (yiyixuxu) [inline docs/source/en/quicktour.mdx:163]: Nice touch to use a cat model example instead of the church! 😸😸😸😸

**Review by yiyixuxu — COMMENTED** [review]
Thank you so much! love how you improved the quicktour with the more comprehensive overview of API; Yet it's still really easy to understand! 

can you guys all take a look? I think this is probably one of the most important page in our doc🤗🤗🤗 @patrickvonplaten @williamberman @pcuenca @patil-suraj @sayakpaul

File: docs/source/en/quicktour.mdx [hunk 4]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+
+* The [`DiffusionPipeline`] is a high-level end-to-end class designed to rapidly generate samples from pretrained diffusion models for inference.
+* Popular pretrained [model](./api/models) architectures and modules that can be used as building blocks for creating diffusion systems.
+* Many different [schedulers](./api/schedulers/overview) - algorithms that control how noise is added - for generating samples from noise during inference and generating noisy images for training.
+
+The quicktour will show you how to use the [`DiffusionPipeline`] for inference, and then walk you through how to combine a model and scheduler to replicate what's happening inside the [`DiffusionPipeline`].
+
+<Tip>
+
+The quicktour is a simplified version of the introductory 🧨 Diffusers [notebook](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/diffusers_intro.ipynb) to help you get started quickly. If you're curious to learn more about 🧨 Diffusers goal, design philosophy, and additional details about it's core API, check out the notebook!
```
[the remark this answers is not in the document]
Comment (sayakpaul) [reply docs/source/en/quicktour.mdx:29]: I think it's okay as a notebook.

File: docs/source/en/quicktour.mdx [hunk 5]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 Before you begin, make sure you have all the necessary libraries installed:
 
 ```bash
 pip install --upgrade diffusers accelerate transformers
 ```
 
-- [`accelerate`](https://huggingface.co/docs/accelerate/index) speeds up model loading for inference and training
-- [`transformers`](https://huggingface.co/docs/transformers/index) is required to run the most popular diffusion models, such as [Stable Diffusion](https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/overview)
+- [🤗 Accelerate](https://huggingface.co/docs/accelerate/index) speeds up model loading for inference and training.
+- [🤗 Transformers](https://huggingface.co/docs/transformers/index) is required to run the most popular diffusion models, such as [Stable Diffusion](https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/overview).
```
Comment (sayakpaul) [inline docs/source/en/quicktour.mdx:40]: Following https://github.com/huggingface/diffusers/pull/2463/files#r1115151675

🤗🤗🤗

File: docs/source/en/quicktour.mdx [hunk 6]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+Diffusion models are trained to denoise random Gaussian noise step-by-step to generate a sample of interest, such as an image or audio. This has sparked a tremendous amount of interest in generative AI, and you have probably seen examples of diffusion generated images on the internet. 🧨 Diffusers is a library aimed at making diffusion models widely accessible to everyone.
+
+Whether you're a developer or an everyday user, this quicktour will introduce you to 🧨 Diffusers and help you get up and generating quickly! There are three main components of the library to know about:
+
+* The [`DiffusionPipeline`] is a high-level end-to-end class designed to rapidly generate samples from pretrained diffusion models for inference.
+* Popular pretrained [model](./api/models) architectures and modules that can be used as building blocks for creating diffusion systems.
+* Many different [schedulers](./api/schedulers/overview) - algorithms that control how noise is added - for generating samples from noise during inference and generating noisy images for training.
```
[the remark this answers is not in the document]
Comment (sayakpaul) [reply docs/source/en/quicktour.mdx:23]: Should we additionally mention that we provide example training scripts that are aimed at education purposes and also for the community to build on top of? It's beautifully explained in the `examples` README: https://github.com/huggingface/diffusers/tree/main/examples#-diffusers-examples

File: docs/source/en/quicktour.mdx [hunk 7]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 | Text-Guided Image-to-Image Translation     | adapt an image guided by a text prompt | [img2img](./using-diffusers/img2img) |
 | Text-Guided Image-Inpainting          | fill the masked part of an image given the image, the mask and a text prompt | [inpaint](./using-diffusers/inpaint) |
 | Text-Guided Depth-to-Image Translation | adapt parts of an image guided by a text prompt while preserving structure via depth estimation | [depth2img](./using-diffusers/depth2img) |
 
-For more in-detail information on how diffusion pipelines function for the different tasks, please have a look at the [**Using Diffusers**](./using-diffusers/overview) section.
+Start by creating an instance of [`DiffusionPipeline`] and specify which pipeline checkpoint you would like to download.
+You can use the [`DiffusionPipeline`] for any [checkpoint](https://huggingface.co/models?library=diffusers&sort=downloads) stored on the Hugging Face Hub.
+In this quicktour, you'll load the [Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion) checkpoint in the [`DiffusionPipeline`] for text-to-image generation.
```
Comment (sayakpaul) [inline docs/source/en/quicktour.mdx:56]:
```suggestion
In this quicktour, you'll load the [Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion) checkpoint in the [`DiffusionPipeline`] for text-to-image generation. Specifically, you will use the [`stable-diffiusion-v1-5`](https://huggingface.co/runwayml/stable-diffusion-v1-5) checkpoint.
```

File: docs/source/en/quicktour.mdx [hunk 8]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
-In this guide though, you'll use [`DiffusionPipeline`] for text-to-image generation with [Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion).
+<Tip warning={true}>
 
-For [Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion), please carefully read its [license](https://huggingface.co/spaces/CompVis/stable-diffusion-license) before running the model.
-This is due to the improved image generation capabilities of the model and the potentially harmful content that could be produced with it.
-Please, head over to your stable diffusion model of choice, *e.g.*  [`runwayml/stable-diffusion-v1-5`](https://huggingface.co/runwayml/stable-diffusion-v1-5), and read the license.
+For [Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion) models, please carefully read the [license](https://huggingface.co/spaces/CompVis/stable-diffusion-license) first before running the model.
+The model's improved image generation capabilities can produce potentially harmful content.
+This quicktour uses the [`stable-diffiusion-v1-5`](https://huggingface.co/runwayml/stable-diffusion-v1-5) checkpoint.
```
Comment (sayakpaul) [inline docs/source/en/quicktour.mdx:62]:
```suggestion
```

File: docs/source/en/quicktour.mdx [hunk 9]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
-You can use the [`DiffusionPipeline`] for any [Diffusers' checkpoint](https://huggingface.co/models?library=diffusers&sort=downloads).
-In this guide though, you'll use [`DiffusionPipeline`] for text-to-image generation with [Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion).
+<Tip warning={true}>
 
-For [Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion), please carefully read its [license](https://huggingface.co/spaces/CompVis/stable-diffusion-license) before running the model.
-This is due to the improved image generation capabilities of the model and the potentially harmful content that could be produced with it.
-Please, head over to your stable diffusion model of choice, *e.g.*  [`runwayml/stable-diffusion-v1-5`](https://huggingface.co/runwayml/stable-diffusion-v1-5), and read the license.
+For [Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion) models, please carefully read the [license](https://huggingface.co/spaces/CompVis/stable-diffusion-license) first before running the model.
+The model's improved image generation capabilities can produce potentially harmful content.
```
Comment (sayakpaul) [inline docs/source/en/quicktour.mdx:61]: Should we mention that to prevent potentially harmful content from being passed to the end user, we use the `safety_checker`?

File: docs/source/en/quicktour.mdx [hunk 10]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+This quicktour uses the [`stable-diffiusion-v1-5`](https://huggingface.co/runwayml/stable-diffusion-v1-5) checkpoint.
 
-You can load the model as follows:
+</Tip>
```
Comment (sayakpaul) [inline docs/source/en/quicktour.mdx:64]: Did not notice the opening tip tag 👁️

File: docs/source/en/quicktour.mdx [hunk 11]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 
-Running the pipeline is then identical to the code above as it's the same model architecture.
+Now you can run the pipeline as you would in the section above.
 
-```python
->>> generator.to("cuda")
->>> image = generator("An image of a squirrel in Picasso style").images[0]
->>> image.save("image_of_squirrel_painting.png")
-```
+### Swapping schedulers
```
[the remark this answers is not in the document]
Comment (sayakpaul) [reply docs/source/en/quicktour.mdx:139]: I don't mind keeping this section here since it's precise. In the dedicated Scheduler section, we dive deeper. So I think it's okay.

File: docs/source/en/quicktour.mdx [hunk 12]
```diff
@@ -10,121 +10,299 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+```py
+>>> with torch.no_grad():
+...     noisy_residual = model(sample=noisy_sample, timestep=2).sample
+```
+
+To generate actual examples though, you'll need a scheduler to guide the denoising process. In the next section, you'll learn how to couple a model with a scheduler.
+
+## Schedulers
+
+Schedulers manage going from a noisy sample to a less noisy sample given the model output - in this case, it is the `noisy_residual`.
```
Comment (sayakpaul) [inline docs/source/en/quicktour.mdx:206]: Should we add a sentence emphasizing that the majority of the pipelines inherited from `DiffusionPipeline` are meant to work with different schedulers we support from the library? 

If we want to expand on it later in the section, I' fine with that.

**Review by sayakpaul — APPROVED** [review]
Amazing, amazing, amazing!

File: docs/source/en/quicktour.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (stevhliu) [reply docs/source/en/quicktour.mdx:23]: Good idea, I added it to the Next Steps section at the end so we don't interrupt the readers flow with too many links! :)

File: docs/source/en/quicktour.mdx (hunk 11 above)
[the remark this answers is not in the document]
Comment (stevhliu) [reply docs/source/en/quicktour.mdx:139]: I think it'd be better here since we're talking about switching components in and out of the pipeline specifically. In the Scheduler section, users are learning more about it in the context of the model and it's a more manual process but it is also more flexible (meaning the user can instantiate whatever scheduler they want). I added a tip to really emphasize that last point, let me know if that's more helpful! :)

File: docs/source/en/quicktour.mdx (hunk 3 above)
Comment (stevhliu) [reply docs/source/en/quicktour.mdx:163]: Haha imo cats are more interesting to look at than churches 😻

File: docs/source/en/quicktour.mdx (hunk 2 above)
Comment (stevhliu) [reply docs/source/en/quicktour.mdx:293]: Hmm I think it'd be a nice for the user to be able to see the final output. In Transformers, I typically save the image output from the notebook and upload it to the documentation-images [repository](https://huggingface.co/datasets/huggingface/documentation-images) where I can embed/link to in the docs.

File: docs/source/en/quicktour.mdx (hunk 9 above)
Comment (stevhliu) [reply docs/source/en/quicktour.mdx:61]: Great idea, I didn't even know this existed!

File: docs/source/en/quicktour.mdx (hunk 12 above)
Comment (stevhliu) [reply docs/source/en/quicktour.mdx:206]: I think we can expand on this later in a different tutorial doc focused on the pipelines. For the quicktour, I think it'd be best to keep the focus on the `DiffusionPipeline` and not really mention the other pipelines to avoid confusion/distracting the user.

File: docs/source/en/quicktour.mdx [hunk 13]
```diff
@@ -10,121 +10,304 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 ```
 
-For more in-detail information on how to change between schedulers, please refer to the [Using Schedulers](./using-diffusers/schedulers) guide.
+Try generating an image with the new scheduler and see if you notice a difference!
+
+In the next section, you'll take a closer look at the components - the model and scheduler - that make up the [`DiffusionPipeline`] and learn how to use these components to generate an image of a cat.
+
+## Models
+
+Models take a noisy sample, and at each timestep it predicts the *noise residual*, the difference between a less noisy image and the input image. This is just one objective though. Other models can make different predictions like the gradient between two timesteps or predict the slightly less noisy image directly. You can mix and match models to create other diffusion systems.
```
Comment (patrickvonplaten) [inline docs/source/en/quicktour.mdx:154]:
```suggestion
Most models take a noisy sample, and at each timestep predicts the *noise residual*, (most because some models are learned to predict the previous sample directly or the velocity/v-objective <link-to-explanation-or-footnote>) the difference between a less noisy image and the input image. This is just one objective though. Other models can make different predictions like the gradient between two timesteps or predict the slightly less noisy image directly. You can mix and match models to create other diffusion systems.
```

While most models predict the noise residual, SD 2.1 doesn't predict the *noise residual* it predicts the velocity (maybe see https://github.com/huggingface/diffusers/issues/1413 and https://github.com/huggingface/diffusers/blob/5e5ce13e2f89ac45a0066cb3f369462a3cf1d9ef/src/diffusers/schedulers/scheduling_ddim.py#L110

File: docs/source/en/quicktour.mdx [hunk 14]
```diff
@@ -10,121 +10,304 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+
+Models are initiated with the [`~ModelMixin.from_pretrained`] method which'll also locally cache the model weights so it is faster the next time you load the model. For the quicktour, you'll load the [`UNet2DModel`], a basic unconditional image generation model with a checkpoint trained on cat images:
```
Comment (patrickvonplaten) [inline docs/source/en/quicktour.mdx:156]:
```suggestion
Models are initiated with the [`~ModelMixin.from_pretrained`] method which will also locally cache the model weights so it is faster the next time you load the model. For the quicktour, you'll load the [`UNet2DModel`], a basic unconditional image generation model with a checkpoint trained on cat images:
```

**Review by patrickvonplaten — APPROVED** [review]
Super nice - only left one nit :-)
